src/nwb_conversion_tools/datainterfaces/ecephys/intan/intandatainterface.py

```diff
@@ -114,43 +129,50 @@ def get_metadata_schema(self):
         return metadata_schema
 
     def get_metadata(self):
-        unique_group_name = set(self.recording_extractor.get_property("group_name"))
+
+        # Add device
+        ecephys_metadata = dict()
         device = dict(
             name="Intan",
             description="Intan recording",
             manufacturer="Intan",
         )
-
         device_list = [device]
+        ecephys_metadata.update(Device=device_list)
 
-        ecephys_metadata = dict(
-            Ecephys=dict(
-                Device=device_list,
-                ElectrodeGroup=[
-                    dict(
-                        name=group_name,
-                        description=f"Group {group_name} electrodes.",
-                        device="Intan",
-                        location="",
-                    )
-                    for group_name in unique_group_name
-                ],
-                Electrodes=[
-                    dict(name="group_name", description="The name of the ElectrodeGroup this electrode is a part of.")
-                ],
-                ElectricalSeries_raw=dict(name="ElectricalSeries_raw", description="Raw acquisition traces."),
+        # Add electrode group
+        unique_group_name = set(self.recording_extractor.get_property("group_name"))
+        electrode_group_list = [
+            dict(
+                name=group_name,
+                description=f"Group {group_name} electrodes.",
+                device="Intan",
+                location="",
             )
+            for group_name in unique_group_name
+        ]
+        ecephys_metadata.update(ElectrodeGroup=electrode_group_list)
+
+        # Add electrodes and electrode groups
+        ecephys_metadata.update(
+            Electrodes=[
+                dict(name="group_name", description="The name of the ElectrodeGroup this electrode is a part of.")
+            ],
+            ElectricalSeries_raw=dict(name="ElectricalSeries_raw", description="Raw acquisition traces."),
         )
 
+        # Add group electrode number if available
         recording_extractor_properties = self.recording_extractor.get_property_keys()
-
         if "group_electrode_number" in recording_extractor_properties:
-            ecephys_metadata["Ecephys"]["Electrodes"].append(
+            ecephys_metadata["Electrodes"].append(
                 dict(name="group_electrode_number", description="0-indexed channel within a group.")
             )
         if "custom_channel_name" in recording_extractor_properties:
-            ecephys_metadata["Ecephys"]["Electrodes"].append(
+            ecephys_metadata["Electrodes"].append(
                 dict(name="custom_channel_name", description="Custom channel name assigned in Intan.")
             )
 
-        return ecephys_metadata
+        # Build metadata dictionary and return complete structure
+        metadata = dict()
```

Comment:
you aren't calling `super().get_metadata`?

Reply:
I just went through the git history of the file and it was never there. Let me check if this allows for code reduction.

Reply:
Added in in the last commit.